web/cypress/support/fba-commands.ts

```diff
@@ -86,8 +92,22 @@ Cypress.Commands.add('setFBAWindSpeed', (windSpeed: string, rowId: number) => {
   return cy.getByTestId(`windSpeedInput-fba-${rowId}`).find('input').type(windSpeed)
 })
 
-Cypress.Commands.add('setDate', (date: string) => {
-  return cy.getByTestId('date-of-interest-picker').find('input').type(date)
+Cypress.Commands.add('setDateTypeMethod', (date: string) => {
+  return cy
+    .getByTestId('date-of-interest-picker')
+    .find('input')
+    .clear()
+    .type(date)
+    .type('{enter}')
+})
+
+Cypress.Commands.add('setDateBlurMethod', (date: string) => {
```

Comment:
This isn't blur is it? This is typing the date, and then click there? A blur would be clicking somewhere else.

Reply:
This clicks off somewhere else because the mouse isn't focused on the input element.